src/ServerSettings.cpp

```diff
@@ -66,11 +66,16 @@ void ServerSettings::parseServerName(const Token value)
 
 void ServerSettings::parseErrorDir(const Token value)
 {
+	if (!_error_dir.empty())
+		throw std::runtime_error("Parsing Error: Redefining error_dir");
```

Comment:
Probably here you only want to specify the type of mistake and later specify whether you handle this as an error or info error warning etc, so in short, "redefining error_dir" would suffice, and letter you would catch it and print it with a logger.log(ERROR or WARNING, e.what() + message)